consensus/polybft/polybft.go

```diff
@@ -779,6 +773,8 @@ func (p *Polybft) SetBlockTime(blockTime time.Duration) {
 	if blockTime >= core.DefaultBaseRoundTimeout {
 		p.ibft.SetBaseRoundTimeout(blockTime * baseRoundTimeoutScaleFactor)
 	}
+
+	p.syncer.UpdateBlockTimeout(blockTime * 3)
```

Comment:
```suggestion
	p.syncer.UpdateBlockTimeout(blockTime * 3)
```
What will happen if blockTime passed to ibft was scaled in the if above, should we have the same blockTimeout as ibft?

Reply:
Hmm, I think we can scale it like the round 0 timeout.

Reply:
Fix: https://github.com/Ethernal-Tech/blade/pull/255/commits/3a2acc5ed48e9a458f4a84979d78e4cb27082642

Reply:
Yes, maybe we need something like this
`scale := 1
if blockTime >= core.DefaultBaseRoundTimeout {
    p.ibft.SetBaseRoundTimeout(blockTime * baseRoundTimeoutScaleFactor)
    scale = baseRoundTimeoutScaleFactor
}

p.syncer.UpdateBlockTimeout(blockTime * 3 * scale)
`